content/en/docs/best-practices/logs-handler-function.md

```diff
@@ -0,0 +1,202 @@
+---
+title: "Logs Handler Function"
+linkTitle: "Logs Handler Function"
+weight: 5400
+description: >	
+    Learn how to use a logs handler function.
+---
+
+This document describes how to use a logs handler function.
```

Comment:
This document describes how to use a logs handler function.
=>
This document describes how to create an async function to find out error logs.

Reply:
[Updated](https://github.com/OpenFunction/website/pull/56/commits/50edb0533d1a32d37524d65b8491c809429fc1bd).